### ISSUE TITLE
Sorting criteria in Source Control pane is not retained

### ISSUE THREAD

**Issue by eldar** [microsoft/vscode#127952]
Issue Type: <b>Bug</b>

Right clincking on the "..." in the Source Control panes brings a context menu that allows to change sorting criteria of files (Sort by Path, Sort by Status, etc). The default order is Sort by Path however I always prefer Sort By Status to see the modified files up top. However this setting is not retained and upon restart of the IDE or reloading of the window the sorting is reverted to Sort by Path, which makes it quite annoying to reset it every time I open a new window.

VS Code version: Code 1.57.1 (507ce72a4466fbb27b715c3722558bb15afa9f48, 2021-06-17T13:26:56.255Z)
OS version: Linux x64 5.4.0-74-generic snap
Restricted Mode: No
Remote OS version: Linux x64 3.10.0-1160.el7.x86_64
Remote OS version: Linux x64 3.10.0-1160.el7.x86_64

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 5 3600 6-Core Processor (12 x 2199)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>rasterization: disabled_software<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|0, 1, 1|
|Memory (System)|31.37GB (15.49GB free)|
|Process Argv|--force-user-env --no-sandbox --unity-launch --no-sandbox --crash-reporter-id a4eb8436-d6c6-4fe2-836f-c669d8bd9def|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|plasma|
|XDG_CURRENT_DESKTOP|KDE|
|XDG_SESSION_DESKTOP|KDE|
|XDG_SESSION_TYPE|x11|

|Item|Value|
|---|---|
|Remote|SSH: gnodea5|
|OS|Linux x64 3.10.0-1160.el7.x86_64|
|CPUs|Intel(R) Xeon(R) CPU E5-2667 v2 @ 3.30GHz (32 x 1440)|
|Memory (System)|251.70GB (236.20GB free)|
|VM|0%|

|Item|Value|
|---|---|
|Remote|SSH: gnodea5|
|OS|Linux x64 3.10.0-1160.el7.x86_64|
|CPUs|Intel(R) Xeon(R) CPU E5-2667 v2 @ 3.30GHz (32 x 1440)|
|Memory (System)|251.70GB (236.20GB free)|
|VM|0%|
</details><details><summary>Extensions (8)</summary>

Extension|Author (truncated)|Version
---|---|---
remote-ssh|ms-|0.65.7
remote-ssh-edit|ms-|0.65.7
remote-wsl|ms-|0.56.5
linux-desktop-file|nic|0.0.14
gitlens|eam|11.5.1
python|ms-|2021.6.944021595
vscode-pylance|ms-|2021.6.3
jupyter|ms-|2021.6.999662501

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
pythonvspyt700cf:30270857
pythonvspyt602:30300191
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30256859
vstes627:30244334
pythonvspyt639:30300192
pythontb:30283811
pythonvspyt551:30311712
vspre833:30321513
pythonptprofiler:30281270
vsdfh931:30280409
vshan820:30294714
pythondataviewer:30285071
vscus158:30321503
pythonvsuse255:30323308
vscorehov:30309549
vscod805cf:30301675
pythonvspyt200:30331937
binariesv615:30325510
vsccppwtct:30329789
aatestflightcf:30331712

```

</details>

**Comment by rabarberski** [comment]
Same issue on macos (with VSC version 1.60.1)

**Comment by BrendanSimon** [comment]
Yes, it's very annoying.  I'm surprised the developers haven't been annoyed with this to have fixed it.
Here's hoping a fix will happen sooner rather than later :)

**Comment by eldar** [comment]
Moreover you have to set it each time for every opened window.
